core/gen_index.ts

```diff
@@ -32,12 +32,23 @@ export function genIndex(base64EncodedConfig: string): string {
     })
     .join("\n");
 
+  const safeIncludeDataformJson = `
```

Comment:
can we instead just do a JSON parse of dataform.json, in native typescript code?